.github/workflows/cargo-checkmate.yaml

```diff
@@ -10,6 +10,10 @@ jobs:
       - name: "Install protoc"
         run: |
           sudo apt-get install -y protobuf-compiler
+      - name: "Setup OPA"
```

Comment:
```suggestion
      - name: "Set up OPA"
```

Reply:
This is intentional, it's the name of the workflow. Sic.

Reply:
Argh, maybe someday then, but not in this PR. Thanks.